website/src/get-started/downloads.md

```diff
@@ -90,6 +90,13 @@ versions denoted `0.x.y`.
 
 ## Releases
 
+## 2.17.0 (2019-12-16)
+Official [source code download](http://www.apache.org/dyn/closer.cgi/beam/2.17.0/apache-beam-2.17.0-source-release.zip).
+[SHA-512](https://www.apache.org/dist/beam/2.17.0/apache-beam-2.17.0-source-release.zip.sha512).
+[signature](https://www.apache.org/dist/beam/2.17.0/apache-beam-2.17.0-source-release.zip.asc).
+
+[Release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12319527&version=12345494).
```

Comment:
The version number is incorrect. I think we should `12345970`(2.17.0) instead of `12345494`(2.16.0).